### ISSUE TITLE
Spark lineage producer always create a new entry instead of updating the existing 

### ISSUE THREAD

**Issue by rajendra-t** [AbsaOSS/spline-getting-started#27]
We have added the producer code below in our Python code and lineage is getting produced and seen in Spline UI. However every time the code is executed it is generating a new entry in the Spline UI. Is there a way to ensure that it overwrites the lineage metadata and shows only latest version of the lineage? 

sc._jvm.za.co.absa.spline.harvester.SparkLineageInitializer.enableLineageTracking(spark._jsparkSession)

![image](https://user-images.githubusercontent.com/99479687/187365275-a7a2b467-f535-4b8d-846f-a54dcced3027.png)

**Comment by cerveada** [comment]
The goal of Spline is to capture all job executions, not just the last one. So it doesn't make sense to overwrite data.

To understand your question a bit better: How do you recognise that two lineages are the different versions of the same lineage and not just two completely different jobs?

**Comment by rajendra-t** [comment]
I agree that the producer will be capturing and appending the lineage information in database, however in UI there should be a way to see the latest lineage based on target node. 

As far as I understand in order to Identify the latest lineage we should refer to the target node and take the latest record. Hence I was looking for a way to only get the latest lineage for the Target dataset.

**Comment by wajda** [comment]
@rajendra-t the screenshot from your post is an event log page where you see all captured execution events. That view is just not designed for grouping per data source. Try the "Data Sources" view (from the main menu on the left hand side). It will show you all captured data sources with information about the latest write.

**Comment by rajendra-t** [comment]
thanks @wajda, I can see the latest lineage in Data Sources tab.
